zookeeper-server/src/main/java/org/apache/zookeeper/cli/SyncCommand.java

```diff
@@ -30,6 +35,7 @@
 
     private static Options options = new Options();
     private String[] args;
+    public static final int CONNECTION_TIMEOUT = 30000;//30s
```

Comment:
What about 'SYNC_TIMEOUT' ?
It is not used for 'comnection'

Reply:
+1

Reply:
Please use TimeUnit so that it is self documenting.

https://docs.oracle.com/javase/7/docs/api/java/util/concurrent/TimeUnit.html

`TimeUnit.SECONDS.toMillis(30L)`